packages/neuron-wallet/src/models/transaction-size.ts

```diff
@@ -28,7 +29,15 @@ export default class TransactionSize {
   }
 
   public static output(output: Output): number {
-    const bytes = serializeOutput(output.toSDK())
+    // const bytes = serializeOutput(output.toSDK())
+    const cellOutput = output.toSDK()
+    const bytes = bytesUtils.hexify(
+      blockchain.CellOutput.pack({
+        capacity: cellOutput.capacity,
+        lock: cellOutput.lock,
+        type: cellOutput.type || undefined, // TODO: null is not a packable type
+      })
+    )
```

Comment:
`cellOutput.type` could be `null`, so maybe it's better to provide a translator to convert `CKBCompont` types with `null`s to `lumos` compatible types with `undefined`s so that the code may look nicer.